src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.exception.AuthenticationException;
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private AuthenticationService authenticationService;
+
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public void register(@RequestBody UserRequestDto userRequestDto) {
+        try {
+            authenticationService.register(userRequestDto.getEmail(), userRequestDto.getPassword());
+        } catch (AuthenticationException e) {
+            throw new RuntimeException("Try to use another email " + e);
```

Comment:
message is duplicated + we should pass the original exception as a parameter and not a string addition to the message
```suggestion
            throw new RuntimeException("Can't register user with email: " + userRequestDto.getEmail(), e);
```